src/views/checkups/self-validation/details/tabs/details/CheckupsSelfValidationDetailsPageSection.tsx

```diff
@@ -0,0 +1,200 @@
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  CardTitle,
+  Grid,
+  GridItem,
+  PageSection,
+  Spinner,
+  Title,
+} from '@patternfly/react-core';
+
+import { STATUS_COMPLETION_TIME_STAMP, STATUS_START_TIME_STAMP } from '../../../../utils/utils';
+import TestStatistics from '../../../components/shared/TestStatistics';
+import {
+  COLOR_FAILED,
+  formatStatusTimestamp,
+  TOTAL_TESTS_FAILED_KEY,
+  TOTAL_TESTS_PASSED_KEY,
+  TOTAL_TESTS_RUN_KEY,
+  TOTAL_TESTS_SKIPPED_KEY,
+} from '../../../utils';
+
+import { ResultsStatus } from './ResultsStatus';
+import TestSuiteCard from './TestSuiteCard';
+
+type CheckupsSelfValidationDetailsPageSectionProps = {
+  configMap: IoK8sApiCoreV1ConfigMap;
+  error: null | string;
+  isJobCompleted?: boolean;
+  job: IoK8sApiBatchV1Job;
+  loading: boolean;
+  results: { [suiteName: string]: any } | null;
+};
+
+const CheckupsSelfValidationDetailsPageSection: FC<
+  CheckupsSelfValidationDetailsPageSectionProps
+> = ({ configMap, error, isJobCompleted, job, loading, results }) => {
+  const { t } = useKubevirtTranslation();
+  const [expandedSections, setExpandedSections] = useState<Record<string, boolean>>({});
+  const isJobFailed = job && job.status?.failed === 1;
+  const toggleSection = useCallback((section: string) => {
+    setExpandedSections((prev) => ({
+      ...prev,
+      [section]: !prev[section],
+    }));
+  }, []);
+
+  const filteredTestSuites = useMemo(() => {
+    if (!results?.tests) return [];
+    return Object.entries(results.tests).filter(([key]) => key !== 'summary');
+  }, [results?.tests]);
+
+  if (loading) {
+    return (
+      <PageSection>
+        <Card>
+          <CardBody>
+            <div style={{ padding: '2rem', textAlign: 'center' }}>
+              <Spinner size="md" />
+            </div>
+          </CardBody>
+        </Card>
+      </PageSection>
+    );
+  }
+
+  if (error) {
+    return (
+      <PageSection>
+        <Card>
+          <CardBody>
+            <div style={{ color: COLOR_FAILED, padding: '2rem', textAlign: 'center' }}>
+              <div>{error}</div>
+            </div>
+          </CardBody>
+        </Card>
+      </PageSection>
+    );
+  }
+
+  if (!configMap) {
+    return (
+      <PageSection>
+        <div>{t('Checkup not found')}</div>
+      </PageSection>
+    );
+  }
+
+  return (
+    <PageSection>
+      <Title className="co-section-heading" headingLevel="h2">
+        {t('Self validation checkup details')}
+      </Title>
+      <Grid>
+        <GridItem span={6}>
+          <div
+            style={{
+              display: 'flex',
+              flexDirection: 'column',
+              gap: 'var(--pf-t--global--spacer--sm)',
+            }}
+          >
+            <div>
+              <strong>{t('Name')}:</strong> {configMap?.metadata?.name}
```

Comment:
The layout looks different then other details pages which are build around [DescriptionList](https://www.patternfly.org/components/description-list) and PatternFly layout components.